services/kubernetesbuilddeploy/src/index.js

```diff
@@ -307,7 +307,7 @@ const messageConsumer = async msg => {
   let namespaceStatus = {}
   try {
     const namespacePost = promisify(kubernetes.namespace.post)
-    namespaceStatus = await namespacePost({ body: {"apiVersion":"v1","kind":"Namespace","metadata":{"name":openshiftProject}} })
+    namespaceStatus = await namespacePost({ body: {"apiVersion":"v1","kind":"Namespace","metadata":{"name":openshiftProject, "labels": {"lagoon/project": projectName}}} })
```

Comment:
we just bought `lagoon.sh` domain, so we can actually use `lagoon.sh` as the label prefix :)
also can we add the environment name in here as well?

Reply:
absolutely.. nice domain! :tada: